src/use_case/TransferPlaylist/TransferPlaylistInteractor.java

```diff
@@ -51,23 +50,22 @@ private void transferFromSpotifyToAmazon(String spotifyAccessToken, String playl
         Playlist spotifyPlaylist = spotifyDAO.getPlaylist(spotifyAccessToken, playlistId);
         List<Song> mappedSongs = mapSongsForAmazon(spotifyPlaylist.getSongs());
         Playlist amazonPlaylist = createAmazonPlaylist(mappedSongs, spotifyPlaylist.getName(), amazonAccessToken);
-        amazonDAO.createPlaylist(amazonPlaylist, amazonAccessToken); // Implement this method in your DAO
+        amazonDAO.createPlaylist(amazonPlaylist, amazonAccessToken); // Implement this method in Amazon Music DAO
     }
 
     // Transfer from Amazon to Spotify
     private void transferFromAmazonToSpotify(String amazonAccessToken, String playlistId, String spotifyAccessToken) throws IOException {
         Playlist amazonPlaylist = amazonDAO.getPlaylist(amazonAccessToken, playlistId);
         List<Song> mappedSongs = mapSongsForSpotify(amazonPlaylist.getSongs());
         Playlist spotifyPlaylist = createSpotifyPlaylist(mappedSongs, amazonPlaylist.getName(), spotifyAccessToken);
-        spotifyDAO.createPlaylist(spotifyPlaylist, spotifyAccessToken); // Implement this method in your DAO
+        spotifyDAO.createPlaylist(spotifyPlaylist, spotifyAccessToken); // Implement this method in Spotify DAO
```

Comment:
Doesn't this require an extra input with a userid as per the method in SpotyifyDataAccessObject? May be bugged otherwise.

Reply:
Good catch! Just fixed up the code to reflect the call to createPlaylist in the transferPlaylist method